packages/@react-spectrum/table/stories/CRUDExample.tsx

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {ActionButton, Button} from '@react-spectrum/button';
+import {ActionGroup} from '@react-spectrum/actiongroup';
+import Add from '@spectrum-icons/workflow/Add';
+import {AlertDialog, Dialog, DialogTrigger} from '@react-spectrum/dialog';
+import {ButtonGroup} from '@react-spectrum/buttongroup';
+import {Cell, Column, Row, Table, TableBody, TableHeader} from '../';
+import {Content} from '@react-spectrum/view';
+import Delete from '@spectrum-icons/workflow/Delete';
+import {Divider} from '@react-spectrum/divider';
+import {Flex} from '@react-spectrum/layout';
+import {Form} from '@react-spectrum/form';
+import {Heading} from '@react-spectrum/typography';
+import {Item, Menu, MenuTrigger} from '@react-spectrum/menu';
+import {Modal} from '@react-spectrum/overlays';
+import More from '@spectrum-icons/workflow/More';
+import React, {useState} from 'react';
+import {TextField} from '@react-spectrum/textfield';
+import {useListData} from '@react-stately/collections';
+
+export function CRUDExample() {
+  let list = useListData({
+    initialItems: [
+      {id: 1, firstName: 'Sam', lastName: 'Smith', birthday: 'May 3'},
+      {id: 2, firstName: 'Julia', lastName: 'Jones', birthday: 'February 10'}
+    ]
+  });
+
+  let [editingItem, setEditingItem] = useState(null);
+  let [deletingItem, setDeletingItem] = useState(null);
+
+  let createItem = (item) => {
+    list.prepend({...item, id: list.items.length + 1});
+  };
+
+  let editItem = (id, item) => {
+    list.update(id, item);
+    setEditingItem(null);
+  };
+
+  let deleteItem = (item) => {
+    list.remove(item.id);
+    setDeletingItem(null);
+  };
+
+  let deleteSelectedItems = () => {
+    list.removeSelectedItems();
+  };
+
+  let onAction = (action, item) => {
+    switch (action) {
+      case 'edit':
+        setEditingItem(item);
+        break;
+      case 'delete':
+        setDeletingItem(item);
+        break;
+    }
+  };
+
+  return (
+    <Flex flexDirection="column">
+      <ActionGroup selectionMode="none" marginBottom={8}>
```

Comment:
TODO: ActionGroup should either be an inline element so alignItems works on the Flex parent to align it to the right, or ActionGroup should support an `align` property. cc. @snowystinger do you have opinions?

Reply:
can't you just set `alignItems="flex-end"` to the <Flex> component?
I don't think ActionGroup is doing anything wrong? having a flex as a parent kinda negates block vs inline. flex has a default alignItems of stretch though, so it might appear as though that's coming into play?

Reply:
`alignItems` isn't exposed to the style props API, so we'd have to add a prop explicitly for ActionGroup...

Reply:
o, no, i meant apply alignItems on Flex

Reply:
ok, `alignItems="flex-end" works. for some reason `alignItems="end"` doesn't.